adding-a-new-scraper.md

```diff
@@ -74,4 +75,5 @@ Please provide documentation on the following:
 2. What fields need to be configured and what they are for.
 3. An example configuration.
 
-This should be provided in a new file under `docs\configuration\metrics`.
+This should be provided in a new file under `docs\configuration\v1.x\metrics`.
```

Comment:
What about this instead of line 79?
```suggestion
This should be provided in a new file under `docs\configuration\v1.x\metrics` and be listed under the supported providers on `docs/configuration/v1.x/metrics/index.md` in alphabetical order.
```